### ISSUE TITLE
UndefinedFunctionError function Qrusty.Native.generate/5 is undefined (module Qrusty.Native is not available

### ISSUE THREAD

**Issue by gushonorato** [nbw/qrusty#9]
Hey,

I'm getting this error in production. 

```
UndefinedFunctionError function Qrusty.Native.generate/5 is undefined (module Qrusty.Native is not available
```

I'm using dokku for my deploys.

Thanks.

**Comment by nbw** [comment]
🤔 thanks! I'll take a look when I have a chance. 

@gushonorato To clarify it's working locally, but not on Dokku? 

Note: might need to update the build targets: https://github.com/nbw/qrusty/blob/main/.github/workflows/release.yml#L27

**Comment by gushonorato** [comment]
@nbw exactly! This error only occurs on Dokku, locally it's working well.
